python/multicorn/__init__.py

```diff
@@ -311,8 +326,15 @@ def execute(self, quals, columns, sortkeys=None):
                 You should return AT LEAST those columns when returning a
                 dict. If returning a sequence, every column from the table
                 should be in the sequence.
+
+        Kwargs:
             sortkeys (list): A list of :class:`SortKey`
                 that the FDW said it can enforce.
+            aggs (dict): A dictionary mapping aggregation key with function and
+                column to be used in the aggregation operation. Result should be
+                returned under the provided aggregation key.
+            group_clauses (list): A list of columns used in GROUP BY statements.
+                The result should be returned for each column name provided.
```

Comment:
What does this mean -- does every row we return need to have an entry for everything in `columns` + `aggs`?

Reply:
What I meant to say is that whenever there is a `group_clauses` kwarg, then for each column specified there the returned response should have a corresponding value for each row using that column name as the key.

I re-worded the docstring as above, hopefully this clarifies it.